Mathlib/LinearAlgebra/QuadraticForm/QuadraticModuleCat.lean

```diff
@@ -0,0 +1,165 @@
+/-
+Copyright (c) 2023 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser
+-/
+import Mathlib.LinearAlgebra.QuadraticForm.IsometryEquiv
+import Mathlib.Algebra.Category.AlgebraCat.Basic
+
+/-!
+# The category of quadratic modules
+-/
+
+open CategoryTheory
+
+universe v u
+
+variable (R : Type u) [CommRing R]
+
+/-- The category of quadratic modules; modules with an associated quadratic form-/
+structure QuadraticModuleCat extends ModuleCat.{v} R where
+  /-- The quadratic form associated with the module. -/
+  form' : QuadraticForm R carrier
```

Comment:
I do not understand the need for `form'`. It seems to me that `form` and `form'` have the exact same types and parameters. I feel that renaming `form'` as `form` and removing some of the definitions below would work fine.

Reply:
Ah, this was a Lean3-ism, where `coeSort` was inserted by the elaborator. I have removed it.